app.js

```diff
@@ -21,78 +21,41 @@ client.login(process.env.DISCORD_TOKEN);
 
 client.on('messageCreate', async function (message)
 {
-    if (message.content.includes('https://twitter.com/'))
-    {        
-        let name = '';
-        let guild = client.guilds.cache.get(message.guildId);
-        guild.members
-            .fetch(message.author)
-            .then(data => name = data.nickname + ' (' + data.user.username + ')');
+    const addressToChangeTwitterTo = "vxtwitter.com";
+    let messageContent = message?.content ?? "";
+    let repostTweet = true;
+
+    // switch to vxtwitter
+    switch(expression) {
+        case messageContent.includes('https://twitter.com/'):
+          messageContent = messageContent.replace('twitter.com', addressToChangeTwitterTo);
+          break;
+        case messageContent.includes('https://x.com/'):
+            messageContent = messageContent.replace('x.com', addressToChangeTwitterTo);
+          break;
+        default:
+            repostTweet = false;
+    }
+
+    // should be a !repost tweet + return statment but I can't test in this environment
+    if(repostTweet) {
 
-        let tweetId = message.content.match(/\/status\/(\d+)/)[1];
-        if (await TweetContainsVideo(tweetId))
-        {
-            message.delete();
-            message.channel.send(name + ' posted: ' + message.content.replace('twitter.com', 'vxtwitter.com'));
+        // strip tracking link
+        if (messageContent.match(/\?t=/gm) != null) {
+            messageContent = messageContent.match(/.+?(?=\?t=)/gm)?.[0] ?? messageContent;
         }
-    }
-    else if (message.content.includes('https://x.com/'))
-    {
-        let name = '';
+
+        // send it cap
         let guild = client.guilds.cache.get(message.guildId);
         guild.members
             .fetch(message.author)
-            .then(data => RepostMessage(message, data.nickname + ' (' + data.user.username + ')', message.content.replace('x.com', 'vxtwitter.com')));
+            .then(data => RepostMessage(message, data.nickname + ' (' + data.user.username + ')', messageContent));
+
```

Comment:
Remove whitespace

Reply:
youre not paying by the kilobyte, it looks nice